README.md

```diff
@@ -100,6 +100,8 @@ API Coverage
 - `partitions`
 - `prod`
 - `rename`
+- `add_prefix`
+- `add_sufix`
```

Comment:
Could we keep this list in alphabetical order for now? :)

Reply:
Changed